documentation/modules/operators/proc-converting-non-managed-topics.adoc

```diff
@@ -0,0 +1,86 @@
+// Module included in the following assemblies:
+//
+// assembly-using-the-topic-operator.adoc
+
+[id='proc-converting-non-managed-topics-{context}']
+= (Preview) Enabling topic management for existing Kafka topics
+
+[role="_abstract"]
+This procedure describes how to enable topic management for topics that are not currently managed through the `KafkaTopic` resource.
+You do this by creating a matching `KafkaTopic` resource.
+
+You can perform this task if you are using unidirectional topic management.
+
+NOTE: Unidirectional topic management is available as a preview. 
+Unidirectional topic management is not enabled by default, so you must xref:ref-operator-unidirectional-topic-operator-feature-gate-{context}[enable the `UnidirectionalTopicOperator` feature gate] to be able to use it.
+
+.Prerequisites
+
+* xref:deploying-cluster-operator-str[The Cluster Operator must be deployed.]
+
+.Procedure
+
+. Create a `KafkaTopic` resource with a `metadata.name` that is the same as the Kafka topic.
```

Comment:
Or use `spec.topicName` is the name of the topic in Kafka would not be a legal Kube resource name.